tests/test_hydromt.py

```diff
@@ -78,7 +78,7 @@ def test_model_build(tmpdir, model):
     mod0 = _model["model"](root=root, mode="r")
     mod0.read()
     # check if equal
-    equal, errors = mod0._test_equal(mod1)
+    equal, errors = mod0._test_equal(mod1, skip_component=["geoms"])
```

Comment:
Should we make a new issue for this to prevent that we forget this fix? ;)

Reply:
Yes!